tests/captures.test.ts

```diff
@@ -19,6 +19,6 @@ describe('gets black and white kill streaks', () => {
   });
 });
 
-describe('tracks kills, deaths and revenge kills by square', () => {
+xdescribe('tracks kills, deaths and revenge kills by square', () => {
```

Comment:
do the tests not work rn, don't remember if I had actually finished this or not

Reply:
haven't tried

Reply:
then why disable

Reply:
nvm see nxt